tests/unit/moduleapi/keyspace_events.tcl

```diff
@@ -67,5 +67,53 @@ tags "modules" {
             assert_equal {1} [r get lua]
             r get x
         } {3}
+
+        test {Test module key space event} {
+            r keyspace.notify x
+            assert_equal {1 x} [r keyspace.is_module_key_notified x]
+        }
+
+        proc __consume_subscribe_messages {client type channels} {
```

Comment:
Maybe we should consider moving this shared code to util.tcl or redis.tcl. 
WDYT?